Better message when `mdbook clean` is run on clean workspace

Currently, if you run `mdbook clean` twice, you get the following error: 

```
$ mdbook clean
2019-10-02 20:18:03 [ERROR] (mdbook::utils): Error: Unable to remove the build directory
2019-10-02 20:18:03 [ERROR] (mdbook::utils):    Caused By: No such file or directory (os error 2)
```

Would it make sense to show a better message to the user when running `mdbook clean` on an already clean workspace? 

If you think this proposal makes sense, I could perhaps do this but I would need suggestions from you. If you think I am missing something and this is not a good idea, please close this issue.

Thank you.

--- a/src/cmd/clean.rs
+++ b/src/cmd/clean.rs
@@ -29,7 +29,10 @@ pub fn execute(args: &ArgMatches) -> mdbook::errors::Result<()> {
         Some(dest_dir) => dest_dir.into(),
         None => book.root.join(&book.config.build.build_dir),
     };
-    fs::remove_dir_all(&dir_to_remove).chain_err(|| "Unable to remove the build directory")?;
+
+    if dir_to_remove.exists() {
+        fs::remove_dir_all(&dir_to_remove).chain_err(|| "Unable to remove the build directory")?;
+    }
 
     Ok(())
 }

--- a/src/cmd/clean.rs
+++ b/src/cmd/clean.rs
@@ -3,7 +3,6 @@ use clap::{App, ArgMatches, SubCommand};
 use mdbook::errors::*;
 use mdbook::MDBook;
 use std::fs;
-use std::path::Path;
 
 // Create clap subcommand arguments
 pub fn make_subcommand<'a, 'b>() -> App<'a, 'b> {
@@ -30,11 +29,7 @@ pub fn execute(args: &ArgMatches) -> mdbook::errors::Result<()> {
         Some(dest_dir) => dest_dir.into(),
         None => book.root.join(&book.config.build.build_dir),
     };
-
-    let path_to_dir_to_remove = Path::new(&dir_to_remove);
-    if path_to_dir_to_remove.exists() {
-        fs::remove_dir_all(&dir_to_remove).chain_err(|| "Unable to remove the build directory")?;
-    }
+    fs::remove_dir_all(&dir_to_remove).chain_err(|| "Unable to remove the build directory")?;
 
     Ok(())
 }

--- a/src/cmd/clean.rs
+++ b/src/cmd/clean.rs
@@ -3,6 +3,7 @@ use clap::{App, ArgMatches, SubCommand};
 use mdbook::errors::*;
 use mdbook::MDBook;
 use std::fs;
+use std::path::Path;
 
 // Create clap subcommand arguments
 pub fn make_subcommand<'a, 'b>() -> App<'a, 'b> {
@@ -29,7 +30,11 @@ pub fn execute(args: &ArgMatches) -> mdbook::errors::Result<()> {
         Some(dest_dir) => dest_dir.into(),
         None => book.root.join(&book.config.build.build_dir),
     };
-    fs::remove_dir_all(&dir_to_remove).chain_err(|| "Unable to remove the build directory")?;
+
+    let path_to_dir_to_remove = Path::new(&dir_to_remove);
+    if path_to_dir_to_remove.exists() {
+        fs::remove_dir_all(&dir_to_remove).chain_err(|| "Unable to remove the build directory")?;
+    }
 
     Ok(())
 }

